docs/pages/accounts/sso.mdx

```diff
@@ -51,9 +51,19 @@ SSO users are like regular users. However, there are a few known exceptions:
 
 ## SSO administration
 
-Both new organizations and existing organizations can enable SSO as a sign in option. Organizations with existing non-SSO members can enable SSO, and both non-SSO users and SSO users can continue to be added as members of SSO organizations.
+Both new organizations and existing organizations can enable SSO as a sign in option. Organizations with existing non-SSO members can enable SSO and then direct new members to the SSO sign-in page, while existing users continue to use their current Expo credentials. To support external contributors, SSO-enabled organizations also support inviting additional non-SSO users via email.
+
+### Transitioning existing users to SSO
+Compared to regular users, SSO users have an exclusive relationship with their organization. Thus, existing users cannot be directly migrated to SSO. However, a user that's already a member of your organization can create a second account by going to the [SSO login page](https://expo.dev/sso-login). Then, their old user can be removed from the organization.
```

Comment:
maybe, not sure about this change, up to you.
```suggestion
Compared to regular users, SSO users have an exclusive relationship with their organization (to maintain security requirements of some organizations). Thus, existing users cannot be directly migrated to SSO. However, a user that's already a member of your organization can create a second account by going to the [SSO login page](https://expo.dev/sso-login). Then, their old user can be removed from the organization.
```

Reply:
I like the idea, not sure about the wording - I just pushed something a little different here. My goal with the paragraph is to echo the restrictions / differences here: https://docs.expo.dev/accounts/sso/#sso-user-restrictions as a reason why the accounts don't lend themselves well to migration. IMO the difference transcends any particular organization's security requirements; org-level ownership of a user just isn't really compatible with the possibility of multi-org access. (I'm thinking of the user who might say, "well, those aren't _my_ requirements, give me the migration!)